src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -117,6 +122,24 @@ public void should_delete_a_Property() {
         assertTrue(properties.isEmpty());
     }
 
+    @Test
+    public void should_update_a_Property() {
+        //Creating
```

Comment:
A good step by step for this test:
- Create an entity to save
- Save the entity 
- Change the fields from the returned entity
- Save the same entity again
- Verify if the fields from the last entity returned are the same as the changed fields